src/community/propose-a-content-change-using-github/index.md.njk

```diff
@@ -0,0 +1,93 @@
+---
+title: Propose a content change using GitHub
+description: Find out how to suggest a change to content in the GOV.UK Design System using GitHub
+section: Community
+layout: layout-pane.njk
+order: 2
+---
+
+The GOV.UK Design System team uses a service called GitHub to manage content in the GOV.UK Design System.
```

Comment:
We usually use 'We' on GOV.UK instead of using our name. Would that be ok to do throughout here?

Reply:
> We are always ‘the GOV.UK Design System team’ in any context, as there are other design system teams across government, and we should make clear that we recognise and respect this.
When talking about the team in the Design System, always refer to us in the third person, and not as ‘we’, ‘us’ or ‘our’. For example, write ‘the backlog is maintained by the Design System team’, not ‘we maintain the backlog’. 
There are other stakeholders involved in the Design System, like the working group and our contributors, and users will not know who ‘we’ is referring to.

Reply:
Yep, what Nick said. So will leave as is for now.